### PR TITLE
Story 701

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: helper/common_handler.js [hunk 1]
```diff
@@ -3116,6 +3116,274 @@ async function ganttChartObject(data) {
...
+
+async function getTaskOverviewChart(tasks) {
+    return new Promise( async function(resolve,reject) {
+
+         let seriesData="[";
+         Object.keys(tasks).forEach( eachTask => {
+            let color = "";
+            if(eachTask=="Completed"){
+                color ="green";
+            }else if(eachTask=="Not Started"){
```
Comment (aks30) [inline helper/common_handler.js:3335]: @rakeshSgr - Is there a space in status?

File: helper/common_handler.js [hunk 2]
```diff
@@ -3038,13 +3038,13 @@ async function ganttChartObject(data) {
                     }
 
                     dataArray.push(obj);
-                } else if (ele.status.toLowerCase() == "not yet started" || ele.status.toLowerCase() == "not started yet") {
+                } else if (ele.status.toLowerCase() == "not yet started" || ele.status.toLowerCase() == "not started yet" || ele.status.toLowerCase() == "notstarted") {
```
Comment (aks30) [inline helper/common_handler.js:3041]: @rakeshSgr - We don't need to handle previous status with space

File: helper/common_handler.js [hunk 3]
```diff
@@ -3116,6 +3116,274 @@ async function ganttChartObject(data) {
...
+                                    var pdfBuffer = Buffer.from(responseHtmlToPdf.body);
+                                    if (responseHtmlToPdf.statusCode == 200) {
+
+                                        fs.writeFile(dir + '/pdfReport.pdf', pdfBuffer, 'binary', function (err) {
+                                            if (err) {
+                                                return console.log(err);
+                                            }
+
+                                            else {
+                                                const s3 = new AWS.S3(config.s3_credentials);
```
Comment (aks30) [inline helper/common_handler.js:3208]: @rakeshSgr - I think this is not required?

Cc - @deepakr123

File: helper/common_handler.js (hunk 2 above)
Comment (rakeshSgr) [reply helper/common_handler.js:3041]: its existing pdf old unnati report code

Comment (aks30) [reply helper/common_handler.js:3041]: @rakeshSgr - But we don't need that right anymore

File: helper/common_handler.js (hunk 1 above)
Comment (rakeshSgr) [reply helper/common_handler.js:3335]: Yes, here we will get status with title case only, as we need to show status like this in pdf

**Pull request merged**